multi_hop_payments test fails

### Background

When I try to run /lnd/lntest/itest/lnd_multi-hop-payments_test.go I get the following error:

```
=== RUN   TestLightningNetworkDaemon
--- FAIL: TestLightningNetworkDaemon (1.59s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x1a942fd]

goroutine 11 [running]:
testing.tRunner.func1.2({0x1c8e9e0, 0x2a242d0})
        /usr/local/go/src/testing/testing.go:1209 +0x24e
testing.tRunner.func1()
        /usr/local/go/src/testing/testing.go:1212 +0x218
panic({0x1c8e9e0, 0x2a242d0})
        /usr/local/go/src/runtime/panic.go:1038 +0x215
github.com/lightningnetwork/lnd/lntest.(*feeService).stop(0x0)
        /Users/wio/Documents/lnd_go/lnd/lntest/fee_service.go:93 +0x1d
github.com/lightningnetwork/lnd/lntest.(*NetworkHarness).Stop(0xc0003325a0)
        /Users/wio/Documents/lnd_go/lnd/lntest/harness_net.go:269 +0x45
github.com/lightningnetwork/lnd/lntest/itest.TestLightningNetworkDaemon(0xc000502b60)
        /Users/wio/Documents/lnd_go/lnd/lntest/itest/lnd_test.go:259 +0xa19
testing.tRunner(0xc000502b60, 0x1fc9418)
        /usr/local/go/src/testing/testing.go:1259 +0x102
created by testing.(*T).Run
        /usr/local/go/src/testing/testing.go:1306 +0x35a
make: *** [itest-only] Error 255
```

### Steps to reproduce

run `make itest icase=multi_hop_payments log=stdout `

### Expected behaviour

I expected the test to pass, since I haven't edited anything. `make itest icase=single_hop_invoice log=stdout` for example worked perfectly 

### Actual behaviour

error above

--- a/lntest/harness_net.go
+++ b/lntest/harness_net.go
@@ -266,7 +266,11 @@ func (n *NetworkHarness) Stop() {
 	close(n.lndErrorChan)
 	n.cancel()
 
-	n.feeService.stop()
+	// feeService may not be created. For instance, running a non-exist
+	// test case.
+	if n.feeService != nil {
+		n.feeService.stop()
+	}
 }
 
 // extraArgsEtcd returns extra args for configuring LND to use an external etcd

